aea/agent.py

```diff
@@ -191,7 +191,12 @@ def start(self) -> None:
 
         :return: None
         """
-        self.runtime.start()
+        was_started = self.runtime.start()
```

Comment:
this PR might be a good chance to update the docstring above? That is:
- Refer to the "runtime" concept
- Remove the breakdown of the function, which I don't think is faithful with respect to the current state.

Reply:
probably best to focus on the code and work on docstring in another PR

Reply:
sure docs  have to be updated, but after code is finalized